sdk/iot/hub/tests/cmocka/test_az_iot_hub_client_c2d.c

```diff
@@ -208,14 +195,11 @@ static void test_az_iot_hub_client_c2d_parse_received_topic_no_props_succeed(voi
   az_iot_hub_client_c2d_request out_request;
 
   assert_int_equal(
-      az_iot_hub_client_c2d_parse_received_topic(&client, test_url_no_props, &out_request),
-      AZ_OK);
+      az_iot_hub_client_c2d_parse_received_topic(&client, test_url_no_props, &out_request), AZ_OK);
 }
 
-static void test_az_iot_hub_client_c2d_parse_received_topic_reject(void** state)
+static void test_az_iot_hub_client_c2d_parse_received_topic_reject()
```

Comment:
I might be mistaken, but isn't the `void**` parameter required for testing without preconditions or filtering test runs? @vhvb1989 - do you remember, or is it fine to remove these parameters?

Reply:
I doubt it - provisioning has the state removed since I first checked it in.